template/scala-akka-http-client/apiInvoker.mustache

```diff
@@ -213,11 +214,22 @@ class ApiInvoker(formats: Formats)(implicit system: ActorSystem) extends CustomC
         Unmarshal(response.entity)
           .to[T]
           .recoverWith {
-            case e ⇒ throw ApiError(response.status.intValue, s"Unable to unmarshall content to [$manifest]", Some(response.entity.toString), e)
+            case e ⇒
```

Comment:
cute, but just for consistency with the code may we replace it to 
```suggestion
            case e =>
```
?

Reply:
sure, it comes from openapi-generator. I didn't noticed the glyph.